test/integration/install-variants_test.js

```diff
@@ -29,7 +31,7 @@ suite('install-variants', function() {
           assert.fail(err);
           done(err);
         }
-        runCommand(binPath, ['install', '--variants', '--offline'], {cwd: tmpPath}).then(() => {
+        runCommand(binPath, ['install', '--variants'], {cwd: tmpPath}).then(() => {
```

Comment:
We do, but I struggled with making this work on travis. It's very strange that it works locally but not on travis, when all of the data should be local.

IMO hermetic tests are better than nonhermetic tests, but continuous integration is better than hermetic tests if you can't have both.